Bridge documentation (#1235)
matrix-org/matrix-appservice-irc: docs/admin_room.md

or to a channel if you have operator priviledges. Note that this command will not produce any response
text, so commands will happen silently.

### `!whois`

`!whois [irc.example.net] NickName|@alice:matrix.org`

A powerful command to either lookup a nickname *or* a Matrix UserID and return information about that user.

### `!storepass`

`!storepass [irc.example.net] passw0rd`

Store a password, or a `username:password` combination to be sent as a PASS command on connection to the server.
This will also reconnect you to IRC so the PASS command can be sent.

**This action will store your password in encrypted form on the IRC bridge**, so be sure to use a unique password
for the IRC service. 

### `!removepass`

`!removepass [irc.example.net]`

Remove your stored password, will NOT reconnect you.

### `!listrooms`

`!listrooms [irc.example.net]`

List all the Matrix rooms that you are joined to which are also connected to IRC.

### `!quit`

QUITs you from all connected networks AND **kicks you from all IRC rooms** (except DMs). This is to avoid
leaking history to your Matrix account while not being visible to IRC users. This command will not remove

> your password or stored data with the bridge. You will need to contact the owner of the bridge to do so.

Reviewer: I find that sentence to be a bit complicated.
```suggestion
your password or stored data with the bridge. Ask the owner of the bridge to remove that data for you.
```

> your password or stored data with the bridge. You will need to contact the owner of the bridge to do so.

Reviewer: Not as part of this PR, but if users can't delete their stored password and data, I think think that's worth an issue/enhancement.

Author: The password can be deleted with the right command, it's just not this command. The data issue is more like "we don't purge all your messages automatically" which is more of a thing we don't do on Matrix as a rule. Maybe I could expand on the latter.

### `!nick`

`!nick [irc.example.net] nick`

Set your nickname for your IRC conenction and persist it across restarts. If the nickname clashes with another
user's nickname, this will fail.

### `!feature`

`!feature feature-name [true/false/default]`

> Enable or disable a feature for your account. Specifying `true` will enable the feature, `false` to disable, or `default`
> to use the default setting based upon the bridge config.

Reviewer: The second sentence switches the tense or something else that's weird.
```suggestion
Enable or disable a feature for your account. Set it to `true` to enable the feature, `false` to disable it, or `default`
to use the default based upon the bridge config.
```

> Currently the featues you can use are:

Reviewer: ```suggestion
Currently, the features you can use are:
```

> - `mentions` - Determine whether IRC users can mention you on the IRC bridge. Not that this will only stop mention text being turned

Reviewer: ```suggestion
- `mentions` - Determine whether IRC users can mention you on the IRC bridge. Note, that this will only stop mention text being turned
```

  into pills. See [this section](usage.md#message-behaviours) for an explanation of this feature.

### `!bridgeversion`

Prints the current version of the bridge.

The bridge bot allows you to control your IRC connection through various bot commands. Some
commands are reserved for bridge administrators, which can be configured in the config file.

### `!plumb`

`!feature !room:example.com irc.network.net #channel`

*This command only works for bridge administrators*

This command allows you to plumb a IRC channel into a room without using the HTTP provisioning API. This command does NOT 
validate that you have permission to do this on the IRC channel so please take care to ensure that the IRC channel is
aware of your actions.

You must invite the bridge bot into the Matrix room for this to work.

### `!help`

> Provides a printout of the bot commands for the bridge

Reviewer: I find these words simpler to understand.
```suggestion
Prints a list of commands for the bridge.
```
